migrate-mongo-config-example.ts

```diff
@@ -1,9 +1,12 @@
 // In this file you can configure migrate-mongo
-
 const config = {
     mongodb: {
+        // Change (or review) the url to your MongoDB:
         url: "MONGODB_URI",
 
+        // Change this to your database name:
+        databaseName: "Aletheia",
```

Comment:
The database name should be present in the mongodb uri, otherwise we can't take advantage of current CI/CD secrets configurations.